README.md

```diff
@@ -1 +1,97 @@
-# capstone-budget-app
\ No newline at end of file
+# Budget App
+
+This a Ruby on Rails based budget app, an interactive project where you can add categories and transactions with user registration and login features.
+
+![imagen](/Screenshot.png)
+
+## Video DEMO
+
+https://www.loom.com/share/2e7beec865ab4f9b9872054b887557bb
+
+## Link Demo
+
+https://floating-brushlands-75570.herokuapp.com/
+
+## Built With
+
+- ![Ruby](https://img.shields.io/badge/Ruby-20232A?style=for-the-badge&logo=ruby&logoColor=61DAFB)
+
+## Getting Started
+
+### Installation
+
+1. Clone the repo
+
+   ```sh
+   git@github.com:miguelesco/recipe_app.git
+   ```
+
+### Usage
+
+1. Install ruby gems dependencies
+
+   ```sh
+   bundle install
+   ```
+
+2. Install node dependencies
+
+   ```sh
+   npm install
+   ```
+
+3. Database creation
+
+   ```
+   rails db:setup
+   ```
+
+4. Run the web server
+
+   ```sh
+   rails server
+   ```
+
+5. Access to the website in a browser with the following link
+
+   ```sh
+   localhost:3000
+   ```
+
+### Test
+
+1. For testing run:
+
+   ```sh
+   rspec spec/* --format doc
+   ```
+
+## Acknowledgements
+
+- Use controllers to handle requests and render empty views.
+- Install Ruby on Rails framework.
+- Use Ruby Gems as a software packages system.
+- Understand Rails naming conventions.
+- Use params from browser requests in a safe way.
+- Write tests for controllers.

```

Comment:
- [ ] The contents that are written under the acknowledgment section have nothing related to the section. Kindly remove unnecessary content here. 
- [ ] Here you should give appropriate credit to Gregoire Vella on Behance the author of the original design, as required by the Creative Commons license of the design.